files/build_templates/qos_config.j2

```diff
@@ -87,6 +87,8 @@
             "7": "7"
         }
     },
+{% elif clear_q_for_pcbb is defined %}
```

Comment:
More changes are needed in this file
1. will need to attach the same scheduler that is used by queues 3 and 4 to the new lossless queus 2 and 6
2. pfc_enable in PORT_QOS_MAP should include the 2 lossless queues for T1 device on the downlinks and T0 device on the uplinks.

Reply:
All addressed. Thanks